Assets/MixedRealityToolkit.Examples/Demos/EyeTracking/DemoScrollPanZoom/Scripts/BaseClasses/PanZoomBase.cs

```diff
@@ -207,7 +207,6 @@ public void AutoPan()
         /// <summary>
         /// Scroll sideways.
         /// </summary>
-        /// <param name="speed"></param>
```

Comment:
Do we want to just remove them or (more work, but) should we actually define them all?

Reply:
I guess, as long as there's some tool that lets us also see where param / return documentation is missing

Reply:
Defining them all would be the best.

I think in order of things of desirability:
1. Having documentation there.
1. Having no empty documentation text.
1. Having no documentation at all.

The reason being, having just empty doc text is the ultimate teaser, and also doesn't actually affect the generated docfx docs (I compared a before/after generated thing and it's the same API docs)

The hope here is that people will just update their things to either delete them (i.e. <returns></returns> where it's obvious what it returns) or update them (param names).

The main goal of this is to move from 3 to 2